kspm-runtime/Chart.yaml

```diff
@@ -39,4 +39,4 @@ dependencies:
   - name: kubeshield-chart
     version: v0.1.2-patch
     repository: oci://public.ecr.aws/k9v9d5v2
-    condition: incluster-scan.enabled
+    condition: inClusterScan.enabled
```

Comment:
Since the ui command is global.inclusterscan.enabled we should make the same here

Reply:
Understood.. Making the changes.

Reply:
Addressed.